python/lsst/utils/tests.py

```diff
@@ -121,13 +122,21 @@ def suiteClassWrapper(tests):
 class MemoryTestCase(unittest.TestCase):
     """Check for resource leaks."""
 
+    # List of regexps to ignore when checking for open files.
+    ignore_regexps = []
```

Comment:
```suggestion
    ignore_regexps: list[str] = []
```
? To make it a bit more explicit when someone looks at the code.